unitTests/ui/MissingTermsTest.ts

```diff
@@ -244,6 +280,41 @@ export function MissingTermsTest() {
             expect(getMissingTerms()).toEqual([KoreanWordPresentMultipleTimes]);
           });
         });
+        describe('when a words contains words boundary character', () => {
+          it(`when the missing term is part of the words,
+          it will not be displayed as a missing term.`, () => {
+            const wordBoundarysCharacter = [
```

Comment:
Rather than duplicating the array, you could use a function to return it. If we ever want to add a new boundary char, there will be one place to update instead of two. e.g.

```
function getWordBoundaryChars() {
    return [...]
}
```